modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -61,6 +61,26 @@ spec:
     maxPods: 500
 ----
 +
+The rate at which the kubelet talks to the API server depends on qps and burst
+values. The default values, `5` for `kubeAPIQPS` and `10` for `kubeAPIBurst`,
+are good enough if there are limited pods running on each node. If there are
+enough CPU and memory resources on the node, then you can update the
```

Comment:
Maybe we should change this to "The default values, `5` for `kubeAPIQPS` and `10` for `kubeAPIBurst`,
are good enough if there are limited pods running on each node but when there are large number of pods with Secrets and ConfigMaps running on each node, updating the kubelet QPS and Burst rates is recommended provided there are enough CPU and memory resources on the node:". Thoughts @ahardin-rh?

Reply:
@chaitanyaenr Thanks! Updated: http://file.rdu.redhat.com/~ahardin/10242019/node-qps-burst/scalability_and_performance/recommended-host-practices.html#create-a-kubeletconfig-crd-to-edit-kubelet-parameters_